<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabSingleImageAnalysis">
       <attribute name="title">
        <string>Single Image Analysis</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4" columnstretch="2,1">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Panel</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0" colspan="2">
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="1">
              <widget class="QRadioButton" name="showExtractedVolRadioBtn">
               <property name="text">
                <string>Show Brain Extraction Mask</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QRadioButton" name="showNormVolRadioBtn">
               <property name="text">
                <string>Show Normalized Volume</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Show Volume:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="showRawVolRadioBtn">
               <property name="text">
                <string>Show Raw Volume</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="3">
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="0" column="4">
                <widget class="QSpinBox" name="numColumnsSpinBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="3">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Number of Columns</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="numRowsSpinBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Number of Rows:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0" colspan="3">
              <widget class="Line" name="line_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <layout class="QGridLayout" name="graphicsViewGridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="sliceXofYLabel">
                  <property name="text">
                   <string>Slice - of -</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSlider" name="slicesSlider">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Control</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="1" column="0" colspan="2">
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="runSpatialNormBtn">
                <property name="text">
                 <string>Run Spatial Norm.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="runBrainExtractionBtn">
                <property name="text">
                 <string>Run Brain Extraction</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="runAnalyzeSelectedPatchBtn">
                <property name="text">
                 <string>Run T.A on 
Selected Patch</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QProgressBar" name="analyzeProgressBar">
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="runAnalyzeWholeSliceBtn">
                <property name="text">
                 <string>Run T.A on
 Whole Slice</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <widget class="Line" name="line_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Texture Analysis Result:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QScrollArea" name="scrollArea">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>367</width>
                <height>342</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QListWidget" name="analyzeResultsListWidget"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="3" column="0">
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="2">
              <widget class="QPushButton" name="exportSingleImgResultsToExcelBtn">
               <property name="text">
                <string>Export All to Excel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="importNIFTIBtn">
               <property name="text">
                <string>Import Raw Brain NIFTI</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Dataset Analysis</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_12" columnstretch="1,1">
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Log</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="0">
            <widget class="QLabel" name="numberOfProcessedFilesLabel">
             <property name="text">
              <string>Processing: - of - files</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QProgressBar" name="datasetAnalysisProgressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Setup</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_8">
             <property name="title">
              <string>Pipeline</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_16" columnstretch="1,1,1,0,1">
              <item row="6" column="0">
               <widget class="QCheckBox" name="pipelineGLSzMcheckBox">
                <property name="text">
                 <string>GLSZM</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>3- Postprocessing Algorithms:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QSpinBox" name="datasetAnalysisNumColsSpinBox">
                <property name="maximum">
                 <number>10000</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="pipelineBrainExbtractioncheckBox">
                <property name="text">
                 <string>Brain Extraction</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QCheckBox" name="pipelineGLRMcheckBox">
                <property name="text">
                 <string>GLRM</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Number of rows:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>2- Texture Analysis Algorithms:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QSpinBox" name="datasetAnalysisNumRowsSpinBox">
                <property name="maximum">
                 <number>10000</number>
                </property>
               </widget>
              </item>
              <item row="0" column="2" colspan="2">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Patches in each slice:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>1- Preprocessing Algorithms:</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QCheckBox" name="pipelineFirstOrderCheckBox">
                <property name="text">
                 <string>First Order</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Number of columns:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" rowspan="9">
               <widget class="Line" name="line_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QCheckBox" name="pipelineGLCMcheckBox">
                <property name="text">
                 <string>GLCM</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="pipelineSpatialNormcheckBox">
                <property name="text">
                 <string>Spatial Normalization</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>Files</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_14">
              <item row="6" column="0" colspan="2">
               <layout class="QGridLayout" name="gridLayout_17">
                <item row="0" column="1">
                 <widget class="QRadioButton" name="saveResultstToDatabaseRadioBtn">
                  <property name="text">
                   <string>Database</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QRadioButton" name="saveResultstToCSVRadioBtn">
                  <property name="text">
                   <string>CSV Files</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QRadioButton" name="saveResultstToBothRadioBtn">
                  <property name="text">
                   <string>Both</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Select directory</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="inputDatasetDirLabel">
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Directory:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Save Mode:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <widget class="Line" name="line_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QPushButton" name="browseOutputDirBtn">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Number of files:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="selectDataDirBtn">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Output Directory:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="startDatasetAnalysisBtn">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Save Directory:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="saveDatasetAnalysisResultsDir">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
